lib/graphql/schema/field.rb

```diff
@@ -91,6 +92,17 @@ def description(text = nil)
         end
       end
 
+      def complexity(new_complexity_proc)
```

Comment:
Integers are also valid for this, right? But I think it will be rejected by this method?

Reply:
Integers are valid even in the block? Alright, adding support and a test case for that.